README.md

```diff
@@ -1,14 +1,50 @@
 # Getting Started with the .NET MAUI ListView 
-A quick-start project that helps you to integrate a ListView control in the .NET MAUI application by adding Syncfusion MAUI ListView package from NuGet. This project also explains on how to select items in the list, change its layout, and add a header and footer.
-
-Refer to the following documentation for the Syncfusion .NET MAUI ListView Control: 
-https://help.syncfusion.com/maui/listview/getting-started
-
-Check out this GitHub example for the Syncfusion .NET MAUI ListView Control: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/ListView
+The [.NET MAUI ListView control](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) is used to present lists of data in a vertical or horizontal orientation with different layouts virtually.A quick-start project that helps you to integrate a ListView control in the .NET MAUI application by adding Syncfusion MAUI ListView package from NuGet. This project also explains on how to select items in the list, change its layout, and add a header and footer.

```

Comment:
Give a single space here:
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-listview/assets/95272441/e4a91e97-8f9e-49fd-b565-f61543913a1a)

Reply:
Added space.